cookiecutter/{{ cookiecutter.package_name }}/e2e/test_template.py

```diff
@@ -24,11 +25,36 @@ def go_to_app(page: Page, streamlit_app: StreamlitRunner):
 
 def test_should_render_template(page: Page):
     frame = page.frame_locator(
-        'iframe[title="my_component\\.my_component"] >> nth=0'
-    )
+        'iframe[title="my_component\\.my_component"]'
+    ).nth(0)
 
     expect(page.get_by_text("You've clicked 0 times!").first).to_be_visible()
 
     frame.get_by_role("button", name="Click me!").click()
 
     expect(page.get_by_text("You've clicked 1 times!").first).to_be_visible()
+
+def test_should_change_iframe_height(page: Page):
+    time.sleep(0.5)
```

Comment:
Can we somehow avoid the sleep operation in E2E tests? Can we wait until some elements are visible on the screen instead of a fixed time value?

Reply:
Done, thanks for suggestion